internal/workflow/activities/bundle.go

```diff
@@ -51,9 +50,7 @@ type BundleActivityParams struct {
 }
 
 type BundleActivityResult struct {
-	RelPath             string // Path of the transfer relative to the transfer directory.
-	FullPath            string // Full path to the transfer in the worker running the session.
-	FullPathBeforeStrip string // Same as FullPath but includes the top-level dir even when stripped.
```

Comment:
I think you could remove the `StripTopLevelDir` parameter after this changes too.

Reply:
I thought I had! :facepalm: :laughing: